sdk/src/test/java/com/google/cloud/dataflow/sdk/util/common/CounterSetTest.java

```diff
@@ -222,4 +226,127 @@ public void testAddCounterMutatorAddMultipleCounters() {
     assertThat(set, containsInAnyOrder(c1, c2));
   }
 
+  @Test
+  public void testNullCounterBeforeAdd() {
+    final String counterName = "counter-name";
```

Comment:
Can make this a static final constant in the class, so it doesn't need to be duplicated in all the tests.